packages/account/PasskeyAccount/passkeyOwnerP256.test.ts

```diff
@@ -8,4 +14,42 @@ describe("PasskeyOwnerP256", () => {
     const signature = await owner.sign(challenge)
     console.log(signature)
   })
+
+  it("should verify signature", async () => {
+    const challenge = "challenge"
+    const signature = await owner.sign(challenge)
+    const signatureDecoded = AbiCoder.defaultAbiCoder().decode(
```

Comment:
nit: Maybe directly give them a name for readability. 

```suggestion
    const [, authenticatorData,, clientDataJSON,,, r, s] = AbiCoder.defaultAbiCoder().decode(
```

Reply:
And in the future, the passkey owner should be refactored into more general shape, which will not be bound to passkey account data structure.